### PR TITLE
Initial checkin

### PR DESCRIPTION
**Pull request by rick-kenna**
Shows Talos Zero Day information

### CODE REVIEW

File: python/vulnerabilities/zero_day_vuln_search.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
+import os
+import sys
+import json
+import requests
+
+VERSION = "1.0.0"
+
+# Process an HTTP error by printing and log.error
+def process_http_error(msg, response, url):
```
Comment (ababhis2) [inline python/vulnerabilities/zero_day_vuln_search.py:9]: log_http_error ?

File: python/vulnerabilities/zero_day_vuln_search.py [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+    vuln_resp = response.json()
+    return vuln_resp['vulnerability']
+
+# Checks if a Talos report URL exists.
+def talos_url_exists(talos_id):
+    talos_vuln_url = f"https://www.talosintelligence.com/vulnerability_reports/{talos_id}"
+
+    try:
+        response = requests.get(talos_vuln_url)
```
Comment (ababhis2) [inline python/vulnerabilities/zero_day_vuln_search.py:46]: should we not be logging talos url does not exist for this talos_id,

Not sure how talos works but doesn't this require API key or some sort of authentication ?

**Review by ababhis2 — APPROVED** [review]
Minor comments

File: python/vulnerabilities/zero_day_vuln_search.py (hunk 2 above)
Comment (rick-kenna) [reply python/vulnerabilities/zero_day_vuln_search.py:46]: Good point.  There is no authentication.  Adding logging is good point.  I can go through and add it.

File: python/vulnerabilities/zero_day_vuln_search.py (hunk 1 above)
Comment (rick-kenna) [reply python/vulnerabilities/zero_day_vuln_search.py:9]: Will add logging.

**Review by xrawlinson — APPROVED** [review]
LGTM!

**Pull request merged**